### PR TITLE
Dactyl manuform 5x6 2 5 keymap

### PR DESCRIPTION
**Pull request by jceb**
## Description

Keymap for a DactylManuform with 5x6 layout, two additional keys (ctrl + alt) and five thumb keys.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/handwired/dactyl_manuform/5x6_2_5/5x6_2_5.h [hunk 1]
```diff
@@ -0,0 +1,27 @@
+#pragma once
```
Comment (drashna) [inline keyboards/handwired/dactyl_manuform/5x6_2_5/5x6_2_5.h:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/handwired/dactyl_manuform/5x6_2_5/5x6_2_5.c [hunk 2]
```diff
@@ -0,0 +1,2 @@
+#include "5x6_2_5.h"
```
Comment (drashna) [inline keyboards/handwired/dactyl_manuform/5x6_2_5/5x6_2_5.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/handwired/dactyl_manuform/5x6_2_5/5x6_2_5.c (hunk 2 above)
Comment (jceb) [reply keyboards/handwired/dactyl_manuform/5x6_2_5/5x6_2_5.c]: Done.

File: keyboards/handwired/dactyl_manuform/5x6_2_5/5x6_2_5.h (hunk 1 above)
Comment (jceb) [reply keyboards/handwired/dactyl_manuform/5x6_2_5/5x6_2_5.h:17]: Done.

**Review by drashna — APPROVED** [review]
LGTM. 

Also, any reason this is targeting develop, rather than master?

**Comment by jceb** [comment]
Nope, nothing in particular. Shall I switch the PR to master?

**Comment by drashna** [comment]
> Nope, nothing in particular. Shall I switch the PR to master?

Ideally, yeah.  It's mostly only core PRs that need to target develop. Unless there is something that is only in develop, you should target master.

**Comment by jceb** [comment]
Okay, I rebased the commits on master.

**Comment by tzarc** [comment]
Sorry about that, GitHub decided to delete the `develop` branch from upstream when we merged it, despite being told otherwise. Reopened.

File: keyboards/handwired/dactyl_manuform/5x6_2_5/readme.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+Make example for this keyboard (after setting up your build environment):
+
+    qmk compile -kb handwired/dactyl_manuform/5x6_2_5 -km default:avrdude-split-right
+    qmk compile -kb handwired/dactyl_manuform/5x6_2_5 -km default:avrdude-split-left
+
+See the [build environment
+setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make
+instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more
+information. Brand new to QMK? Start with our [Complete Newbs
+Guide](https://docs.qmk.fm/#/newbs).
```
Comment (fauxpark) [inline keyboards/handwired/dactyl_manuform/5x6_2_5/readme.md:27]:
```suggestion
The Dactyl ManuForm Mini is a fork of the [Dactyl ManuForm](https://github.com/tshort/dactyl-keyboard). The Dactyl ManuForm is a fork of the [Dactyl](https://github.com/adereth/dactyl-keyboard) with the thumb cluster from [ManuForm](https://github.com/jeffgran/ManuForm).

This [specific version](https://github.com/jceb/dactyl-manuform-mini-keyboard) of the Dactyl ManuForm Mini adds to additional keys to each side, CTRL and ALT, that are missing in the default configuration of the mini keyboard. In addition, a thumb cluster with 5 keys is enabled.

The keyboard is custom built and handwired. There multiple build guides available.

Make example for this keyboard (after setting up your build environment):

    make handwired/dactyl_manuform/5x6_2_5:default

See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by ridingqwerty — COMMENTED** [review]
This looks good to me, and fauxpark's request looks like it was addressed.

**Review by ridingqwerty — APPROVED** [review]

**Pull request merged**